applications/MeshingApplication/custom_utilities/tetrahedra10_mesh_converter_utility.h

```diff
@@ -0,0 +1,356 @@
+// KRATOS  __  __ _____ ____  _   _ ___ _   _  ____
+//        |  \/  | ____/ ___|| | | |_ _| \ | |/ ___|
+//        | |\/| |  _| \___ \| |_| || ||  \| | |  _
+//        | |  | | |___ ___) |  _  || || |\  | |_| |
+//        |_|  |_|_____|____/|_| |_|___|_| \_|\____| APPLICATION
+//
+//  License:		 BSD License
+//                                       Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ariadna Cortés
+//
+
+#if !defined(KRATOS_TETRAHEDRA10_MESH_CONVERTER_UTILITY)
+#define  KRATOS_TETRAHEDRA10_MESH_CONVERTER_UTILITY
+
+// System includes
+
+/* Project includes */
+#include "custom_utilities/local_refine_tetrahedra_mesh.hpp"
+#include "geometries/tetrahedra_3d_10.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+///@}
+///@name Type Definitions
+///@{
+///@}
+///@name  Enum's
+///@{
+///@}
+///@name  Functions
+///@{
+///@}
+///@name Kratos Classes
+///@{
+class Tetrahedra10MeshConverter : public LocalRefineTetrahedraMesh
+{
+public:
+
+    ///@name Type Definitions
+    ///@{
+    ///@}
+
+    typedef Node<3> NodeType;
+    typedef Geometry<NodeType> GeometryType;
+    typedef GeometryType::Pointer GeometryPtrType;
+
+    ///@name Life Cycle
+    ///@{
+
+    /// Default constructors
+    Tetrahedra10MeshConverter(ModelPart& model_part) : LocalRefineTetrahedraMesh(model_part)
```

Comment:
Styling, method arguments should be CamelCase. It is repeated in many places.

Reply:
[Here](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide) you can find the style guide.